applications/serial_lte_modem/doc/TCPUDP_AT_commands.rst

```diff
@@ -684,11 +690,17 @@ Syntax
 
 ::
 
-   #XUDPSEND[=<data>]
+   #XUDPSEND=[<data>][,<flags>]
 
 * The ``<data>`` parameter is a string that contains the data to be sent.
   The maximum size of the data is 1024 bytes.
   When the parameter is not specified, SLM enters ``slm_data_mode``.
+* The ``<flags>`` parameter sets the sending behavior. It can be set to the following value:
+
+  * ``512`` Blocking send operation until the request is acknowledged.
+    The request will not return until the send operation is completed by lower layers,
+    or until the timeout, given by the AT_SO_SNDTIMEO socket option, is reached.
+    Valid timeout values are 1 to 600 seconds.
```

Comment:
```suggestion
* The ``<flags>`` parameter sets the sending behavior.
  It can be set to the following value:

  * ``512`` - Blocks send operation until the request is acknowledged.
    The request will not return until the send operation is completed by lower layers, or until the timeout given by the AT_SO_SNDTIMEO socket option, is reached.
    Valid timeout values are 1 to 600 seconds.
```